exercises/Data-frames-1-R.md

```diff
@@ -15,14 +15,7 @@ their volumes (`l * w * h`). You could do this using a spreadsheet, but the
 project that you are working on is going to be generating lots of these files so
 you decide to write a program to automate the process.
 
-Download the data, use `read.csv()` to import it into R, and then print out:
+Download the data, use `read.csv()` to import it into R, and then use the `$` operator to print out:
```

Comment:
What is the use of `$` for here? I think I was envisioning just printing out the entire data frame.